spec/requests/api/users_spec.rb

```diff
@@ -1,6 +1,6 @@
 require "rails_helper"
 
-describe "Users API" do
+context "Users API" do
   describe "GET/users/:id" do
```

Comment:
It should be "describe", but Rubocop is causing issues because it wants top level describe to involve a class name, not a string. I'll try and find a solution for it.